### ISSUE TITLE
[question] Does tianshou automatically log metrics in WandB?

### ISSUE THREAD

**Issue by arvganesh** [aai-institute/tianshou#836]
- [ ] I have marked all applicable categories:
    + [ ] exception-raising bug
    + [ ] RL algorithm bug
    + [ ] documentation request (i.e. "X is missing from the documentation.")
    + [ ] new feature request
- [x] I have visited the [source website](https://github.com/thu-ml/tianshou/)
- [x] I have searched through the [issue tracker](https://github.com/thu-ml/tianshou/issues) for duplicates
- [x] I have mentioned version numbers, operating system and environment, where applicable:
  ```python
  import tianshou, gymnasium as gym, torch, numpy, sys
  print(tianshou.__version__, gym.__version__, torch.__version__, numpy.__version__, sys.version, sys.platform)
  ```
Version Details:
```
0.5.0 0.27.1 1.13.1.post200 1.23.5 3.8.16 | packaged by conda-forge | (default, Feb  1 2023, 16:01:55) 
[GCC 11.3.0] linux
```

I'm unsure how to log specific metrics using Tianshou and can't find many examples or tutorials explaining how to do so. Thanks!

I'm using the `onpolicy_trainer` with `PPOPolicy` in a custom `gymnasium` environment.

*Second Question*: Additionally, I'm not seeing any output in my terminal updating me on the progress of my training. What should I do about this? Is there flag I need to turn on? [Here](https://github.com/arvganesh/example-retro-integration/blob/main/PPO_agent.py) is my training script for further details, I based it off of the ppo example for atari.

**Comment by Trinkle23897** [comment]
If you use WandbLogger, the metric would be automatically updated to wandb. See https://tianshou.readthedocs.io/en/master/tutorials/logger.html

**Comment by arvganesh** [comment]
Thanks for the quick response! If I don't see metrics logged after a certain amount of time, does that mean that not enough epochs have passed for training metrics to be logged? Do you have any thoughts on the 2nd question at the bottom of the post? @Trinkle23897

**Comment by Trinkle23897** [comment]
not sure what's going on, can you first test with this script to see if it successfully logs things to wandb? https://github.com/thu-ml/tianshou/blob/d5d521b3295e716eca67c49295bb4db8c5b47a2a/test/modelbased/test_psrl.py#L88

then follow the implementation in `test_psrl.py`, set a very small log interval (default is 1000) in your script, to see if anything is logged into wandb

**Comment by arvganesh** [comment]
I ended up figuring it out, realized I had a bug in my training script. Thanks for your help!
